.github/workflows/main.yml

```diff
@@ -41,6 +41,11 @@ jobs:
           echo 'gem: --no-document' | sudo tee /etc/gemrc
           cd _build
           bundle install
+          git clone -b 2.5.2 --depth 1 https://github.com/pygments/pygments.git
+          git clone -b master --depth 1 https://github.com/nikeee/pygments-lexer-graphviz.git
+          cd pygments-lexer-graphviz/
+          python setup.py develop
+          cd ../
```

Comment:
Something here seems to break GH Actions build (may be the very first thing, calling `git`?). I suggest that we'll extract it into separate step to avoid confusion with installation of Ruby dependencies, or at least rename the step.

Reply:
Yeah, thanks for the catch, it was missing a `sudo`. In https://github.com/kaitai-io/kaitai_struct_formats/pull/278/commits/00963931d294e7331fca9cf2241ceade85ffa11f, it finally works.